api/internal/git/repospec.go

```diff
@@ -214,29 +212,25 @@ func parseHostSpec(n string) (string, string) {
 			host += p
 		}
 	}
-	// ssh relative path
 	if host == "git@" {
-		var i int
-		// git orgRepo delimiter for ssh relative path
-		j := strings.Index(n, ":")
-		// orgRepo delimiter only valid if not preceded by /
-		k := strings.Index(n, "/")
-		// orgRepo delimiter exists, so extend host
-		if j > -1 && (k == -1 || k > j) {
-			i = j
-			// should try to group / with previous component
-			// this is possible if git url username expansion follows
-			if k == i+1 {
-				i = k
-			}
+		i := strings.Index(n, "/")
+		if i > -1 {
 			host += n[:i+1]
 			n = n[i+1:]
+		} else {
+			i = strings.Index(n, ":")
+			if i > -1 {
+				host += n[:i+1]
+				n = n[i+1:]
+			}
 		}
 		return host, n
 	}
 
+	// TODO(annasong): replace this array with Schemes
 	// If host is a http(s) or ssh URL, grab the domain part.
-	for _, p := range Schemes() {
+	for _, p := range []string{
+		"ssh://", "https://", "http://"} {
```

Comment:
This revert should include removing `Schemes` (L23) because it was introduced for this change

Reply:
`Schemes` should be treated the same as `HasRemoteFileScheme` in fileloader.go. They are additions that don't affect the code path loading remotes.

Reply:
Are you using it anywhere else or planning to use it in a future PR?

I want to make sure we are not keeping around code that isn’t or won’t be used.